types/jsftp-lsr/jsftp-lsr-tests.ts

```diff
@@ -0,0 +1,16 @@
+import jsftp from 'jsftp';
+import jsftpLsr from 'jsftp-lsr';
```

Comment:
Please use `import jsftpLst = require('jsftp-lsr')` syntax to match the commonjs-style `export = JsftpLsr`. 

The documentation for jsftp uses `var jsftp = require('jsftp')`, so it, too, should use `import jsftp = require('jsftp')`.

Reply:
I've updated the code according to your suggestions. Also, I've made changes to `jsftp` typings, in order to avoid:
```
import jsftp = require('jsftp');
const ftp = new jsftp.default();
```
Also, `jsftp` typings should now be more compliant with the [class template](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-class-d-ts.html).